src/main/java/com/github/princesslana/eriscasper/repository/event/ChannelsFromEvents.java

```diff
@@ -31,12 +32,13 @@
   private static final ChannelFunctionData<Guild> ADD_GUILD_CHANNELS_FUNCTION =
       ChannelFunctionData.of(
           (map, guild) -> {
-            if (guild.getChannels().isEmpty()) {
+            if (guild.getChannels().orElse(ImmutableList.of()).isEmpty()) {
```

Comment:
Is this the right approach?

If it's not present, don't we want to that that as 'no updates'?

Reply:
If it's not present it will represent an empty list which has no updates. It works as you've explained.